composer.json

```diff
@@ -64,5 +64,15 @@
                 ".htaccess": ".htaccess"
             }
         }
-    }    
+    },
+    "scripts": {
+        "post-create-project-cmd": [
+            "@init-styleguide"
+        ],
+        "init-styleguide": [
+            "[[ ! -d styleguide ]] && git clone git@github.com:palantirnet/palantir-lab.git styleguide || true",
+            "rm -rf styleguide/.git/ styleguide/LICENSE",
```

Comment:
I don't know if removing the license is allowed, since it's the MIT license. However, the palantir-lab styleguide is intended as a template, and in our use we want to be able to include it under the licensing/copyright of the project we use it within.

Reply:
We would have to remove the license from the base project, I think. I suspect keeping this portion as MIT is ok.